City-of-Helsinki/drupal-helfi-sote
# UHF-8238: Disable old IBM Watson chatbots, enable new version.

# [UHF-8238](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8238)

## What was done
This PR disables old IBM Watson chatbot embeds and enables the new one.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-8238-new-ibm-chatbots`
  * `make fresh`
* Run `make drush-cr`

## How to test
  * Can't be fully tested yet because the applications need to be updated on IBM's side first
* [ ] Compare the [new block config](https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/admin/structure/block) to the disabled "ChatLeijuke" blocks (the non disabled ones such as KUURA and SUUNTE should not be affected) and the Platform config libraries file: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/blob/3.x/helfi_platform_config.libraries.yml
  * The pages the block appears on should be the same (note that the default embed should have the "Hide for the listed pages" setting enabled)
  * The tentantID and assistandID parameters should be the same  
  * Check the YML files and make sure there's nothing weird there

## Designers review

## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/426
* https://github.com/City-of-Helsinki/drupal-helfi-asuminen/pull/222

[UHF-8238]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
